src/BenchmarkDotNet/Loggers/AsyncProcessOutputReader.cs

```diff
@@ -89,13 +95,21 @@ private void Detach()
         private void ProcessOnOutputDataReceived(object sender, DataReceivedEventArgs e)
         {
             if (!string.IsNullOrEmpty(e.Data))
+            {
                 output.Enqueue(e.Data);
+                if (logOutput)
+                    logger.WriteLineInfo(e.Data);
```

Comment:
If we use `WriteLine` we don't get the Info color:

Before:

![image](https://user-images.githubusercontent.com/6011991/158963025-b3b0ac2d-9bda-45e5-84ea-443f3f485fcb.png)

After:

![image](https://user-images.githubusercontent.com/6011991/158963087-76672ea2-972d-4501-bd25-c1e16576aa29.png)


```suggestion
                    logger.WriteLine(e.Data);
```